viya4-iac-aws-darksite/README.md

```diff
@@ -0,0 +1,39 @@
+# Using the `viya4-iac-aws` GitHub project in a Dark Site Deployment Scenario (Experimental)
+
+### Contributors
+
+We thank the following individuals for technical assistance and their contributions for the documentation, scripts and yaml templates that provided the basis for this document.
+- Josh Coburn
+- Matthias Ender
+
+### Background for Dark Site Deployment Scenarios
+
+This documentation contains procedures that can be used to successfully complete a Dark Site deployment using elements of `viya4-iac-aws` GitHub project decribed below.
+
+#### Dark Site Deployment Target Recommendation
+**Note:** _It is recommended for a Dark Site deployment that you use a [BYON](https://github.com/sassoftware/viya4-iac-aws/blob/main/docs/user/BYOnetwork.md) or Bring Your Own Network configuration (scenario 3)  with IAC by creating your own VPC, subnets, AWS private endpoints, and network security groups versus letting `viya4-iac-aws` create these items for you.  If you rely on `viya4-iac-aws` to create the network infrastructure in a Dark Site scenario, you will likely run into failure points.  The procedures listed below assume that you use a BYON configuration scenario with IAC._
```

Comment:
Not sure why we would recommend scenario #3.  Scenario #2 is preferred provided they set the *.private_access_cidrs variables.  Scenario #3 now exists primarily to support some edge case (yet to be seen) where IaC can't create the SG rules needed in support of Scenario #2.